#205 Wrong downloaded torrent file name

mario-nt opened the pull request:
Fixes #205

mario-nt commented:
[torrust/torrust-index-types-lib](https://github.com/torrust/torrust-index-types-lib) Needs to be updated, and package.json has to reflect that change before submitting the PR for review.

da2ce7 commented on package.json:
```diff
@@ -43,4 +43,4 @@
     "torrust-index-types-lib": "^0.2.0",
     "uuid": "^9.0.0"
   }
-}
```
no need to edit this file (removing the trailing newline)

mario-nt replied:
Fixed with new commit.

josecelano replied:
Hi @mario-nt, in general, I prefer to rewrite the PR commits instead of revert changes if they are not merged yet. Just to have the history as clean and straightforward as possible.

mario-nt replied:
Hi @josecelano, I thought about that too and wasn't sure about wich approach to take. I will go and modify the commits as I think too, it creates a clearer commit history.

Commit:
fix: [#205] downloaded torrent file's name is the title not the name

Now, when the user downloads a torrent file, that file is named after the torrent name
instead of the torrent title.

The parameter torrent.title was replaced by torrent.name in every call to the function
that handles the download of the torrent file.

Commit:
build: [#205] downloaded torrent file's name is the title not the name

Updated packages so now they contain the name of the torrent in the
responses from the API calls and the typescript types.

mario-nt commented:
Rebase and tests OK @josecelano, Dependencies updated too.

josecelano commented:
> Rebase and tests OK @josecelano, Dependencies updated too.

Cool! Thank you!

josecelano commented:
ACK c097875c999085b871e9947286d4bc2124667458
